client/js/app/utils/ExplorerUtils.js

```diff
@@ -79,36 +82,34 @@ module.exports = {
     }
     var params = _.cloneDeep(explorer.query);
 
-    // Set the timeframe (will get removed if it's null o undefined)
-    params.timeframe = module.exports.getTimeframe(explorer);
```

Comment:
We set the timeframe on the explorer query to `null` when we move over to a `funnel` analysis type, so this function call will fail. We'll need `queryJSON` to take into account whether we're dealing with a `funnel` query or not and adjust what we try to parse/add to the params.